content/docs/design/conditional-branch/conditional-branch.md

```diff
@@ -0,0 +1,178 @@
++++
+title = "Conditional Branch Requirement Overview and Design"
+description = "This page provides an overview of `Conditional Branch Requirement`"
+weight = 10
+toc = true
+aliases = ["/docs/design/"]
+[menu.main]
+  parent = "Design"
+  weight = 2 
++++
+
+Kubegene needs if/else conditional branch and switch/case conditional branch handling.
```

Comment:
personally I think we should first introduce the background usecase, and then  elaborate this feature

Reply:
yeah,changed